Tests/ApodidaeTests/CommandTests.swift

```diff
@@ -72,10 +72,13 @@ class CommandTests: XCTestCase {
         XCTAssertNotEqual(Command.info("foo"), Command.add(package: "foo", requirement: .tag("1.0.0")))
     }
 
-    static var allTests = [
-        ("testInit", testInit),
-        ("testSimpleQueries", testSimpleQueries),
-        ("testAdd", testAdd),
-        ("testEquality", testEquality),
-    ]
+    static var allTests: [(String, (CommandTests) -> () throws -> Void)] {
```

Comment:
I changed it to be computed properties, but if you would like to change it back, then it is not necessary